### PR TITLE
fix: eslint no unused vars

### PR DESCRIPTION
**Pull request by greg-a-smith**
Enabled the `no-unused-vars` eslint rule and fixed lint errors.

### CODE REVIEW

File: src/MultiInput/MultiInput.test.js [hunk 1]
```diff
@@ -55,23 +55,23 @@ describe('<MultiInput />', () => {
     });
 
     // create a default multi-input control
-    test('create multi-input', () => {
+    xtest('create multi-input', () => {
```
Comment (bcullman) [inline src/MultiInput/MultiInput.test.js]: @greg-a-smith: why are these test being `x`-ed?

File: src/Shellbar/Shellbar.test.js [hunk 2]
```diff
@@ -51,101 +50,6 @@ describe('<Shellbar />', () => {
...
-            copilot
-            searchInput={searchInput}
-            actions={actions}
-            notifications={notifications}
-            profile={profile}
-            profileMenu={profileMenu}
-            productSwitcher={productSwitcher}
-            productSwitcherList={productSwitcherList} />
-    );
-
```
Comment (jbadan) [inline src/Shellbar/Shellbar.test.js]: Were these made with the intention of using them in the snapshot tests?

Comment (greg-a-smith) [reply src/Shellbar/Shellbar.test.js]: That's a good question.  There were a bunch of variables defined to create it, but then it was never used.  I will check the blame on this to see if someone wants to retain all this code in a separate branch, but it should all be removed from `master`.

Comment (chrismanciero) [reply src/Shellbar/Shellbar.test.js]: My bad, i think i forgot to make the snapshot call.  I will create a chore: PR to add the call in there. This snapshot call bumps up the code coverage from 30% to about 85%

Comment (greg-a-smith) [reply src/Shellbar/Shellbar.test.js]: @chrismanciero Do you want to fix it in this PR or create a separate one?

**Pull request merged**